src/traces/scattersmith/attributes.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var hovertemplateAttrs = require('../../plots/template_attributes').hovertemplateAttrs;
+var texttemplateAttrs = require('../../plots/template_attributes').texttemplateAttrs;
+var extendFlat = require('../../lib/extend').extendFlat;
+var scatterAttrs = require('../scatter/attributes');
+var lineAttrs = scatterAttrs.line;
+
+module.exports = {
+    mode: scatterAttrs.mode,
+
+    re: {
+        valType: 'data_array',
+        editType: 'calc+clearAxisTypes',
+        description: 'Sets the real coordinates'
+    },
+
+    im: {
+        valType: 'data_array',
+        editType: 'calc+clearAxisTypes',
+        description: 'Sets the imaginary coordinates'
+    },
+
+    text: scatterAttrs.text,
+    texttemplate: texttemplateAttrs({editType: 'plot'}, {
+        keys: ['re', 'im', 'text']
+    }),
+    hovertext: scatterAttrs.hovertext,
+
+    line: {
+        color: lineAttrs.color,
+        width: lineAttrs.width,
+        dash: lineAttrs.dash,
+        shape: extendFlat({}, lineAttrs.shape, {
+            values: ['linear', 'spline']
+        }),
+        smoothing: lineAttrs.smoothing,
+        editType: 'calc'
+    },
+    connectgaps: scatterAttrs.connectgaps,
+
+    marker: scatterAttrs.marker,
+    cliponaxis: extendFlat({}, scatterAttrs.cliponaxis, {dflt: false}),
+
+    textposition: scatterAttrs.textposition,
+    textfont: scatterAttrs.textfont,
+
+    fill: extendFlat({}, scatterAttrs.fill, {
```

Comment:
Do we have tests for `fill`?

Reply:
No, but my understanding is that fill doesn't really make sense in a smith chart. Should I keep it and write a test, or remove the attribute?

Reply:
Nonetheless, it does seem to work, and I added a test: 383b6f147e23ee62e88370fd5e9344eabe629fbf